pkg/agent/controller/egress/egress_controller_test.go

```diff
@@ -70,6 +70,21 @@ func (d *fakeLocalIPDetector) HasSynced() bool {
 	return true
 }
 
+type fakeIPAssigner struct {
+}
+
+func (a *fakeIPAssigner) AssignEgressIP(egressIP, egressName string) error {
+	return nil
+}
+
+func (a *fakeIPAssigner) UnassignEgressIP(egressName string) error {
+	return nil
+}
+
+func (a *fakeIPAssigner) AssignedIPs() (ips map[string]string) {
+	return
+}
+
```

Comment:
why aren't we mocking the `IPAssigner` interface instead, this is typically what we do for this project?

Reply:
done

Reply:
I found that the Go import lines generated by mockgen:
```
import (
	gomock "github.com/golang/mock/gomock"
	reflect "reflect"
)
```
Is this typically?
We should use Goimports grouping imports?
```
import (
	reflect "reflect"

	gomock "github.com/golang/mock/gomock"
)
```

Reply:
The only exception for code style is auto-generated code. We don't touch it anyway, no point to unify them. And code-generators may have their own templates that cannot unify.